### ISSUE TITLE
Upgrade KFP Istio configuration to v1.6

### ISSUE THREAD

**Issue by james-jwu** [kubeflow/manifests#1707]
Currently KFP uses Istio RBAC policies that is going to be deprecated in v1.6. The policies needs to be upgraded to supported Istio 1.6 in future KF releases.

**Comment by yanniszark** [comment]
@james-jwu I think this is a pipelines issue and the KFP manifests will live in https://github.com/kubeflow/pipelines.
Should we close this issue for https://github.com/kubeflow/pipelines/issues/4768?

**Comment by james-jwu** [comment]
The changes need to be made to files under https://github.com/**kubeflow/manifests**/tree/master/pipeline/installs/multi-user. That's why I opened a new issue under the kubeflow/manifests repo. IIUC the issue needs to be in the same repo as the affected files.

**Comment by yanniszark** [comment]
> IIUC the issue needs to be in the same repo as the affected files.

Exactly!

However, as part of the wg manifests effort, the manifest development should happen in the upstream app repo. We will come back soon with an issue for next actions on this plan. I will close this issue once we have the plan. 😄

**Comment by james-jwu** [comment]
I am mostly done with the writing and testing fix, and almost ready to submit the PR. Do you want to put this work on hold until the manifests are moved to respective repos?

**Comment by Bobgy** [comment]
Let me clarify two points:
1.
> IIUC the issue needs to be in the same repo as the affected files.

I don't think that's necessary, it's also OK to write a PR in kubeflow/manifests repo and claim it was written because of an issue in kubeflow/pipelines repo.

2.
@yanniszark I'll be working on moving KFP manifests (especially multi-user manifests) back to KFP repo. I think @james-jwu is already mostly done with his development. We can let that in and move after that.

**Comment by yanniszark** [comment]
> I think @james-jwu is already mostly done with his development. Shall we let his change in? and I will move them out of manifests repo in one shot.

Yes of course, I wasn't aware that this was a done job. I agree with merging the update now and moving it out later.

**Comment by yanniszark** [comment]
@james-jwu @Bobgy I understand this is complete right?
Should we close this issue?

**Comment by Bobgy** [comment]
It should be, but we haven't finished verifying it in distribution, because we were blocked on profile controller.

/assign @zijianjoy 
Is working on verification

**Comment by Bobgy** [comment]
With manifest rc0 cut, we can mark this as closed.
